### DISCUSSION TITLE
How to set a 'Teams' Theme and 'strong' scheme on v8 ThemeProvider?

### DISCUSSION THREAD

**Discussion by MrCodeWeaver** [microsoft/fluentui#16343] [Q&A] [answered]
I am using ThemeProvider in my app by importing ThemeProvider from '@fluentui/react-theme-provider';

I want to be able to set the app theme to be 'Teams' Theme and the Scheme to be 'strong', like you can see in the documentation here: https://fabricweb.z5.web.core.windows.net/oufr/8.0.0-beta.2/#/examples/themeprovider#Variants 
![image](https://user-images.githubusercontent.com/12052413/103558796-2c8c3300-4e83-11eb-87b9-9e96387611a5.png)

In those examples you can select a Theme and a Scheme.  But I don't see in the code for those how the Theme and Scheme are being set.
How do I set the Theme and the Scheme for the ThemeProvider?

**Comment by JustSlone** [comment, marked as answer]
As far as I know there isn't an official Teams them for @fluentui/react-theme-provider. 

@xugao might be able to help out with more info on what theme is being used there in the documentation.
